src/ui/components/CardSlider/CardSlider.scss

```diff
@@ -0,0 +1,35 @@
+.card-slider {
+  .card-slider-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 1rem;
+  }
+
+  .pagination {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0.25rem 0;
+  }
+
+  .page-indicator,
+  .page-indicator-active {
+    margin: 0 0.25rem;
+    background: var(--ion-color-dark-grey);
+    opacity: 0.5;
+    width: 1rem;
+    height: 1rem;
+    border-radius: 0.5rem;
+  }
+
+  .page-indicator-active {
+    width: 30px !important;
```

Comment:
That would be `1.875rem`.

Any reason for using pixels instead?

Can you also add a media query for small screens? (Usually we do `value * 0.8` in this case `24px` which is `1.5rem`).

Reply:
Sry, I copied css from `Slides` component and forgot check.